silk/tests/silk_replay.py

```diff
@@ -237,7 +246,7 @@ def run(self):
             time.sleep(delay)
           self.execute_message(entity_name, message, timestamp)
 
-    self.output_summary()
+    self.output_summary(True)
```

Comment:
```suggestion
    self.output_summary(coalesced=True)
```

Unclear what `True` means without argument name?